intelmq/bots/outputs/elastic/output.py

```diff
@@ -54,15 +54,15 @@ def process(self):
         for field in self.flatten_fields:
             if field in event_dict:
                 val = event_dict[field]
-                # if it string try to convert to json
+                # if it's a string try to parse it as JSON
                 if isinstance(val, str):
                     try:
                         val = loads(val)
-                    except:
+                    except ValueError:
                         pass
                 if isinstance(val, Mapping):
                     for key, value in val.items():
-                        event_dict[key] = value
+                        event_dict[field + '_' + key] = value
```

Comment:
I was thinking of taking a different approach viz allowing the option to overwrite keys or not.  If `overwrite_keys` is `True` then skip the key while flattening else overwrite. It is a matter of choice though, however the reason I favour this is purely aesthetic i.e a field named `time_observation` vs `extra_time_observation`.

Overwriting, at first thought, sounds like a bad idea however, considering most of the non-extra fields will anyways be filled with proper values, only extra data which does not fits into existing harmanization will be flattened, as such the probability of overwriting actually being hit will be very less, correct me if i am missing something.